README.md

```diff
@@ -136,6 +136,84 @@ custom:
   datadog:
     enabled: ${self:custom.staged.dd_enabled, true}
 ```
+### Serverless Monitors 
+
+Serverless has seven recommended monitors with default values pre-configured. 
```

Comment:
I would reword this... not sure it really makes sense to say that "Serverless" has recommended monitors.